js/components/SpidLoginButton.js

```diff
@@ -441,5 +454,12 @@ const styles = StyleSheet.create({
   spinner: {
     width: 20,
     marginRight: 6
+  },
+  errorMissingToken: {
+    padding: 5,
+    backgroundColor: '#F23333',
```

Comment:
are we going to refactor color codes somewhere in a single place? or are we going to wait the moment when the designer says "I'd like to change that color everywhere"? :D

Reply:
Maybe we don't have to deal with styles at all in this development phase, just use defaults ?

Reply:
waiting the final graphic, i add it in common style (components/style,js) (errorContent)